README.md

```diff
@@ -76,9 +81,9 @@ Below are the common issues you may face when building this repository's documen
     ```
   The table of contents is embedded into each topic with this template which increases the build time and HTML file sizes. Alternatively, you can override the browser's restrictions (which may be insecure). For example, Google Chrome and Microsoft Edge accept the `--allow-file-access-from-files` command line switch which allows loading local files.
   
-  > We recommend using the `--serve` DocFX switch to preview documentation, and then share it with end users via a web server, instead of browsing the file system 
+  > We recommend using the `--serve` DocFX switch to preview documentation, and then share it with end users via a web server instead of browsing the file system.
 
-If your issue is not listed, do not hesitate to [submit a new issue to this repository](https://github.com/DevExpress/dotnet-eud/issues/new) or contact us using the [DevExpress Support Center](https://www.devexpress.com/Support/Center/). You can search the [DocFX issues list](https://github.com/dotnet/docfx/issues), or try building the [docfx\-seed](https://github.com/docascode/docfx-seed) sample documentation project to check if your issue is specific to this repository. 
+If your issue is not listed, you can [submit a new issue to this repository](https://github.com/DevExpress/dotnet-eud/issues/new) or contact us via the [DevExpress Support Center](https://www.devexpress.com/Support/Center/). You can search the [DocFX issues list](https://github.com/dotnet/docfx/issues), or try building the [docfx\-seed](https://github.com/docascode/docfx-seed) sample documentation project to check if your issue is specific to this repository. 
 
-## Obtain End-User Documentation for Versions Prior to 17.1
+## Obtain End-User Documentation for Versions Before 17.1
```

Comment:
Why is this information at the end?

Reply:
Because old versions are rarely used.

Reply:
Perhaps it is more beneficial to add it to the start? Something like Note: If you require documentation for v17.1 and earlier...

Reply:
...you can download end-user documentation for previous versions in CHM and PDF formats from https://www.devexpress.com/Support/Documentation/download.xml?platform=user-dev-docs?